cleverhans/utils.py

```diff
@@ -220,6 +220,95 @@ def grid_visual(data):
     return figure
 
 
+def get_logits_over_interval(sess, model, x, adv_x, x_data,
+                             min_epsilon=-10, max_epsilon=10,
+                             num_points=21):
+    """Get logits when the input is perturbed in an interval in adv direction.
+
+    Args:
+        sess: Tf session
+        model: Model for which we wish to get logits
+        x: Tf placeholder for data
+        adv_x_real: Tf placeholder for adv perturbation of x
```

Comment:
The parameter is called `adv_x` but the docstring calls it `adv_x_real`

Reply:
Fixed.